### ISSUE TITLE
Project Page: Embedded-GUI Player is too small (screens < 1096px)

### ISSUE THREAD

**Issue by LiFaytheGoblin** [LLK/scratch-www#2084]
Starting on 1095px the player/stage gets small - it should always fill out the same width as the green-flag-bar. This is because gui is implemented here, it sets the width somehow.

<img width="1001" alt="screen shot 2018-09-13 at 1 58 49 pm" src="https://user-images.githubusercontent.com/5471273/45512043-96d9b080-b76c-11e8-99fb-dd5de4e77c1a.png">

### Operating System and Browser
MacOS, Chrome

**Comment by benjiwheeler** [comment]
Yes, that's why -- gui's layout-constants file defines the sizes, and then `components/gui.jsx` uses `react-responsive`'s `MediaQuery` together with the `getStageDimensions` and `resolveStageSize` functions in `screen-utils` to decide what size to show.

Seems like we could pass info from www about how the player is embedded, and use that to show alternate sizes.

**Comment by benjiwheeler** [comment]
(Moved this into a github project)

**Comment by benjiwheeler** [comment]
(Moved this into a github project)
